scripts/windows/validate.ps1

```diff
@@ -1,32 +1,17 @@
 #Requires -Version 5.0
```

Comment:
honestly, no reason to even run validate on windows. could drop this file entirely

Reply:
Dropped